### PR TITLE
Update commands and README.md

### PR DESCRIPTION
**Pull request by benvand**
* Update dev commands to compile to correct place, remove nodemon in favour of node-sass --watch, update package.lock
* Update README.md to reflect current commands and structure 
* Remove incorrectly committed file

### CODE REVIEW

**Comment by UI-TEC** [comment]
Good to go

**Review by kypky — COMMENTED** [review]
The changes look great! Good to go!

**Review by cevangelougovcy — COMMENTED** [review]
Also looks good

**Review by UI-TEC — APPROVED** [review]
Nice work Ben, I've no issues

File: package.json [hunk 1]
```diff
@@ -6,20 +6,19 @@
...
         "css-start": "npm run css-watch",
-        "watch-css": "nodemon -e scss -x \"npm run css-build\"",
+        "watch-css": "npm run css-watch",
         "start": "http-server -c-1 -p 3000 -o build/"
     },
     "dependencies": {
         "bootstrap": "^5.1.3"
     },
     "devDependencies": {
+        "copyfiles": "^2.4.1",
```
Comment (benvand) [inline package.json:20]: I think copyfiles is a bad dependency to have

File: package.json [hunk 2]
```diff
@@ -6,20 +6,19 @@
     "author": "DMRID - DSF Team",
     "license": "MIT",
     "scripts": {
-        "css-build": "node-sass --omit-source-map-url src/sass/main.scss build/css/main.css && copyfiles src/js/app.js build/js ",
+        "css-build": "copyfiles -u 2 src/js/app.js build/js && node-sass --omit-source-map-url src/sass/main.scss build/css/main.css",
```
Comment (benvand) [inline package.json:9]: YThis command is cross platform now 👍

**Pull request merged**